recipes-mender/mender/files/mender.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Mender OTA update service
+
+[Service]
+Type=simple
+User=root
+Group=root
+ExecStart=/usr/bin/mender
```

Comment:
Yup. Since we don't have it right now systemd will just fail starting Mender but this is enough for now to continue with Mender Go implementation.